experimental/PropertyDDS/packages/property-proxy/package.json

```diff
@@ -16,7 +16,7 @@
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
     "build:commonjs": "npm run tsc && npm run build:test",
     "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
-    "build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+    "build:docs": "",
```

Comment:
can we just delete it for now?

Reply:
or open an issue for it to fix and put in something similar to what is in `merge-tree`?

Reply:
I would prefer to just remove this as well.